packages/commons/src/file-manager/fileManager.ts

```diff
@@ -98,6 +113,28 @@ export function initFileManager(
         throw fileManagerCopyError(filePathToCopy, key, bucket, error);
       }
     },
+    get: async (
+      bucket: string,
+      path: string,
+      logger: Logger
+    ): Promise<string> => {
+      logger.info(`Getting file ${path} in bucket ${bucket}`);
+      try {
+        const response = await client.send(
+          new GetObjectCommand({
+            Bucket: bucket,
+            Key: path,
+          })
+        );
+        const body = response.Body;
+        if (!body) {
+          throw fileManagerGetError(bucket, path, "File is empty");
+        }
+        return await streamToString(body as Readable);
+      } catch (error) {
+        throw fileManagerGetError(bucket, path, error);
+      }
+    },
```

Comment:
there are situations where we want to have the original stream. Personally, I prefer to have a dedicated function to convert the file stream to a string

Reply:
Changed [here](https://github.com/pagopa/interop-be-monorepo/pull/663/commits/8fe83a71e2446f02735440c80624532c86bbdac2)